pom.xml

```diff
@@ -32,7 +32,7 @@
         <munit.citizen.input.directory>src/test/munit-citizen</munit.citizen.input.directory>
         <munit.output.directory>${basedir}/target/test-mule/munit</munit.output.directory>
         <munit.extensions.maven.plugin.version>1.4.0</munit.extensions.maven.plugin.version>
-        <munit.version>3.3.1</munit.version>
+        <munit.version>3.3.2</munit.version>
         <mtf.tools.version>1.2.0-rc1</mtf.tools.version>
```

Comment:
There is a final version for this [here](https://github.com/mulesoft/mtf-tools/tags)

Reply:
Done